### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by ObjectIsAdvantag**
Hi Clement,
As proposed during your Human Talk last tuesday, here it comes : tests and sample now run on Windows (and WebStorm).
Nothing broken hopefully on linux (as Travis build is OK, https://travis-ci.org/ObjectIsAdvantag/cliparse-node/builds)
Regards
Stève

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+.idea/
```
Comment (divarvel) [inline .gitignore:2]: I prefer to not have editor-specific stuff in .gitignore

File: src/cli.js [hunk 2]
```diff
@@ -116,7 +116,8 @@ cli.parse = function(cliApp, argv) {
   var options = _.omit(cliValues, "_");
   var args = cliValues._;
 
-  if(args[0] === "node") {
+  // check the command is launch via the node interpreter (+ ensure windows compat)
```
Comment (divarvel) [inline src/cli.js:119]: launched\* :-)

**Comment by divarvel** [comment]
Nice! minor nitpicking in the comments :-)

**Comment by divarvel** [comment]
I have integrated the code and fixed a few things. It is available in the 0.2.1 release.
Thanks :-)

**Comment by ObjectIsAdvantag** [comment]
Hello Clement, 
I just checked,
Your commit works ok but not in all scenarios.
let me explain : simplifying the check with === 'node.exe' won't make it when people specify the whole path to the node executable, 
for example on windows with WebStorm, args[0] in cli.js returns "C:\Program Files\nodejs\node.exe", the purpose of the regexp is to handle this scenario.
This cloud also be handful on linux / WebStorm configs I guess.
Hope it helps...

**Comment by divarvel** [comment]
I'll extract the code in a separate function so you can add tests for these
cases

Regards,
Clément Delafargue

PGP Fingerprint : 5311 A6C4 8416 5378 3019  B977 FFB4 9299 EAE0 ED53

**Comment by divarvel** [comment]
https://github.com/divarvel/cliparse-node/commit/8809bf6ee37db6c22717e4fdd08ed2acabec9f77

**Comment by ObjectIsAdvantag** [comment]
test ok on Windows 7 64 bits, node 0.10.35

**Pull request merged**